roles/openshift_node/files/bootstrap.yml

```diff
@@ -61,6 +61,7 @@
       with_items:
       - line: "BOOTSTRAP_CONFIG_NAME=node-config-{{ openshift_group_type }}"
         regexp: "^BOOTSTRAP_CONFIG_NAME=.*"
+      no_log: True
```

Comment:
Is this one sensitive?

Reply:
I'll test and find out. I was probably erring on the side of too many no_log statements.